cou/steps/plan.py

```diff
@@ -129,13 +129,26 @@ async def _generate_data_plane_plan(
     hypervisor_apps, non_hypervisors_apps = _separate_hypervisors_apps(
         analysis_result.apps_data_plane
     )
+    plans = []
+    if args.upgrade_group != HYPERVISORS:
+        plans.extend(
+            _generate_data_plane_subordinate_remaining_plan(
+                target, non_hypervisors_apps, args.force
+            )
+        )
```

Comment:
I guess this is not ideal anymore. We'll always need to upgrade subordinates before the hypervisors. This can be moved to the code bellow.